src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -34,6 +34,7 @@ function SearchPageBottomTab() {
         return {
             queryJSON: buildSearchQueryJSON(searchParams.q, searchParams.policyIDs),
             policyIDs: searchParams.policyIDs,
+            isSearchResultsMode: searchParams.isCustomQuery !== 'false',
```

Comment:
NAB, not very important, but I'd move it out to a helper function even if it's a 1-liner.